index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Damon Romano's.js Quiz</title>
+  </head>
+  <body>
+
+    How many lines tall would you like your pine tree to be?
+    <input type="text" name="Rows?" autofocus id="pineTreeHeight">
+
+    Please type the single character that will comprise your Pine Tree.
+    <input type="text" maxlength="1" name="Character?" id="pineTreeCharacter">
```

Comment:
Name attribute removed.

At one point, I thought the name attribute was necessary to make the input boxes display as I wanted them to. I cannot now explain why I ever thought that.